commit 8f7d3133302354145ed9e4b13eecce48549a90c6
Author: leo.singer@LIGO.ORG <leo.singer@LIGO.ORG@ac820499-f4a4-4ea9-b4e2-a7172f5148fc>
Date:   Wed Sep 11 21:39:38 2013 +0000

    Don't use LINKFORSHARED; it's not for building Python extension modules
    
    git-svn-id: https://svn.ligo.caltech.edu/svn/nds2/trunk@1156 ac820499-f4a4-4ea9-b4e2-a7172f5148fc

diff --git config/ax_python_devel.m4 config/ax_python_devel.m4
index cf2163c..469b36b 100644
--- config/ax_python_devel.m4
+++ config/ax_python_devel.m4
@@ -41,6 +41,10 @@
 #   Copyright (c) 2009 Horst Knorr <hk_classes@knoda.org>
 #   Copyright (c) 2013 Daniel Mullner <muellner@math.stanford.edu>
 #
+#   MODIFIED 2013-09-11 by Leo Singer <leo.singer@ligo.org>: don't use
+#   LINKFORSHARED; according to <http://trac.macports.org/ticket/39223>,
+#   LINKFORSHARED is not needed for building Python extension modules.
+#
 #   This program is free software: you can redistribute it and/or modify it
 #   under the terms of the GNU General Public License as published by the
 #   Free Software Foundation, either version 3 of the License, or (at your
@@ -271,18 +275,6 @@ EOD`
 	AC_SUBST(PYTHON_EXTRA_LIBS)
 
 	#
-	# linking flags needed when embedding
-	#
-	AC_MSG_CHECKING(python extra linking flags)
-	if test -z "$PYTHON_EXTRA_LDFLAGS"; then
-		PYTHON_EXTRA_LDFLAGS=`$PYTHON -c "import distutils.sysconfig; \
-			conf = distutils.sysconfig.get_config_var; \
-			print (conf('LINKFORSHARED'))"`
-	fi
-	AC_MSG_RESULT([$PYTHON_EXTRA_LDFLAGS])
-	AC_SUBST(PYTHON_EXTRA_LDFLAGS)
-
-	#
 	# final check to see if everything compiles alright
 	#
 	AC_MSG_CHECKING([consistency of all components of python development environment])
